modules/protocol_gateways/setup_s3.sh

```diff
@@ -0,0 +1,60 @@
+echo "$(date -u): running S3 script"
+
+wait_for_weka_fs || exit 1
```

Comment:
should be in the validation script

Reply:
But for smb we check smbw first 

`if [[ ${smbw_enabled} == true ]]; then
  wait_for_weka_fs || exit 1
  create_config_fs || exit 1
fi`

Reply:
you echo in the validation:
`echo "$(date -u): Done running validation"` which isn't true because it isn't completed.
You can have a flag: `fs_validation` which will be True for s3 and smbw.